examples/patterns/instantiate/structured_config/my_app.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from dataclasses import dataclass, field
+from typing import Any, List
+
+from omegaconf import MISSING, DictConfig
+
+import hydra
+from hydra.core.config_store import ConfigStore
+from hydra.types import ObjectConf
+from hydra.utils import instantiate
+
+
+class DBConnection:
+    def __init__(self, host: str, port: int) -> None:
+        self.driver = "driver"
+        self.host = host
+        self.port = port
+
+    def connect(self) -> None:
+        print(f"{self.driver} connecting to {self.host} on port={self.port}")
```

Comment:
```suggestion
        print(f"{self.driver} connecting to {self.host}:{self.port}")
```

And fix corresponding tests and doc examples.